### DISCUSSION TITLE
广义瓦片经济 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#476] [General]
# 广义瓦片经济 - Miao Yu | 于淼

https://yufree.cn/cn/2024/09/11/general-annuity/

**Comment by tcgriffith** [comment]
这一篇特别有经济思维，可惜现实世界的运转不只看经济。

美国2024年平均每天1.5起大规模枪击案，受害人两万多，川普几个月内被暗杀至少两次， 波音吹哨人死于非命。 以上这些社会矛盾发生在人均GDP85373美元的国家。说明人均收入水平再高也不妨碍基层组织能力和社会治理水平堕落成shithole country。

> 到一定程度打工人会发现自己再怎么辛苦劳动也无法进入瓦片阶层，此时就会质疑当前社会体制，引发轻一点的改革或重一点的战争。

在米帝金融霸权的框架下，即使是最底层也享受着世界财富集中后  涓滴效应层层渗透下来的汤汤水水，物质水平让90%的世界人民望尘莫及。 想靠打工人就引发社会变革是不是想多了…

**Reply by yufree** [reply]
我感觉说的是一件事，国家人均这两个词凑一块在当前贫富差距与货币实际购买力下已经基本没啥参考价值了，内部矛盾不解决只能转外部树立共同敌人，而所谓内部矛盾的解决几乎等同于内战实现财富分配重制，现实中都是用渐进式政策来缓和这个过程，例如延迟退休。
我说的打工人不考虑国籍的，我认识美国三班倒的工人，关心的也是账单与柴米油盐，一月能搞出接近1000刀的电费，也骂垄断企业不是人，也为找不到工作的子女跟前妻赡养费头痛，也数着日子等退休吃社保，超过90%世界人民生活水平跟他们没啥关系，也是被平均的。当然美国产业工人确实不多，不过服务业能赚多的也不开放给低学历。但他们也没法跟其他国家的工人或底层服务业共情，TikTok或脸书里又不存在，抖音跟B站也更喜欢新闻过度解读与大国叙事，真正的矛盾都在菜市场与超市里，只是爆发时会被重定义为一个高尚的目标。

**Comment by earfanfan** [comment]
>不过这三档参保人数分配极为畸形，第一档只有五千多万，第二档大概五亿，第三档五亿，但看近年总开支上来说，第二档比第三档高了十倍有余，跟第一档持平，从人均上就会出现100:10:1这样的离谱分配结构。

我今天看到一个比较具体的数据，来源于[成武县2023年国民经济和社会发展统计公报](http://www.chengwu.gov.cn/10086/8a1af4577d6741fa017d763b1e830012/1783057580797644800.html)。

>社会保障能力稳步提高。2023年机关事业单位退休人员8778人，发放养老待遇7.85亿元；企业单位离退休人员13220人，发放养老待遇3.69亿元；城乡居民养老保险待遇领取人员11.94万人，发放养老待遇2.51亿元；

**Reply by earfanfan** [reply]
之前看了三水你这篇文章后，我恰好又看到了楼上的数据，再后来我发现官方的数据有些删减，再再后来我就写了[看一看我国机关事业单位的养老标准](https://yuanfan.rbind.io/project/red-and-black/)，主要是收集了一些公布了机关事业单位养老保险的县级公报。

再再再后来……下面的推算是我5天前写的。

人大常委执法检查组出了一份报告，人大官网有一则[我国社会保险工作取得历史性成就](http://www.npc.gov.cn/npc/c2/c30834/202411/t20241107_440659.html)，其中有一段文字是这样写的。

>与此同时，我国社会保险保障水平也在不断提升。据统计，2012年至2023年，企业退休人员月人均养老金由1686元提高到3162元，城乡居民月人均养老金由82元提高到214元，失业保险金由707元提高到1814元，工伤保险月人均伤残津贴由1864元提高到4051元。

如果把2012、2022、2023年的人力资源与社会保障统计公报里与城镇职工基本养老保险的一些指标摘出来，只得到下面的数据。其中2022年城镇职工中企业的在职缴费和离退休人数数据源于国家统计局。

|年份|类别|参保人数（万人）|缴费人数（万人）|离退休人数（万人）|基金收入（亿元）|基金支出（亿元）|月人均养老金（元）|
|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
|2023|城镇职工基本养老保险|52121|37925|14196|70506|63757||
|2023|城镇职工-企业|46044|||||3162|
|2023|城镇职工-机关事业单位|||||||
|2022|城镇职工基本养老保险|50355|36711|13644|63323.8|59034.7||
|2022|城镇职工-企业||32817.5|11530.9||||
|2022|城镇职工-机关事业单位|||||||
|2012|城镇职工基本养老保险|30427|22981|7446|20001|15562||
|2012|城镇职工-企业|28272||6804.6|||1686|
|2012|城镇职工-机关事业单位|||||||

接下来，我要做一些非常粗糙的推算，主要目的是推算机关事业单位的月人均养老金上涨了多少。

其一，根据“2012年至2023年，企业退休人员月人均养老金由1686元提高到3162元”这句话，假设这11年来企业范围内的月人均养老金是匀速上涨，那么2022年的数值就是3028。

其二，缴纳城镇职工养老保险的除了企业、机关事业单位，应当还有一些灵活就业人员，但灵活就业人员的数据从未公布，因此本次推算假定城镇职工养老保险的数据中减去企业得到的就是机关事业单位的数据。

其三，这些指标之间应当有一些换算公式，参保人数=缴费人数+离退休人数，月人均养老金=(基金支出/离退休人数)/12。

那么接下来会推算出来这样的结果。

|年份|类别|参保人数（万人）|缴费人数（万人）|离退休人数（万人）|基金收入（亿元）|基金支出（亿元）|月人均养老金（元）|
|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
|2023|城镇职工基本养老保险|52121|37925|14196|70506|63757|3742.7|
|2023|城镇职工-企业|46044|||||3162|
|2023|城镇职工-机关事业单位|6077||||||
|2022|城镇职工基本养老保险|50355|36711|13644|63323.8|59034.7|3605.7|
|2022|城镇职工-企业|44348.4|32817.5|11530.9||41898.7|3028|
|2022|城镇职工-机关事业单位|6006.6|3893.5|2113.1||17136|6757.8|
|2012|城镇职工基本养老保险|30427|22981|7446|20001|15562|1741.7|
|2012|城镇职工-企业|28272|21467.4|6804.6||13767.1|1686|
|2012|城镇职工-机关事业单位|2155|1513.6|641.4||1794.9|2332|

根据上表推算的结果可以得到这样的一个结论：从2012年到2022年，机关事业单位的月人均养老金从2332元上涨到了6757.8元，企业的月人均养老金从1686元上涨到了3028元。在2012年，机关事业单位的月人均养老金是企业的1.38倍，而到2022年扩大到了2.23倍。

我之前在统计那些县级公报数据的时候是手动整理的，不得不阅读很多公报的内容，会看到有些县提到城乡居民养老保险、城镇职工养老保险、机关事业单位的人均养老金上涨了多少，很明显的就是，在上涨的金额数字上是 机关事业单位>城镇职工>城乡居民。三水你的文章提到“未来大概率会两头改革”，不知道你当时写的时候所想的未来是撒时候呢？我所猜想的是，并不只是整个离退休人员是一个巨大的利益共同体，而是整个机关事业单位的在职缴费和离退休人员都是一个巨大的利益共同体，即便中央政府基于社会稳定等多方面考量希望更平均一些，可是真到了基层政府，财政支出的权利是掌握在地方政府手里的，政府内部就会有上与下的对抗，自然就会造成机关事业单位的退休待遇涨的更多了。

**Reply by yufree** [reply]
现在在推机关事业单位与企业职工退休金并轨，不过执行层面大概率是新人新办法老人老办法的和稀泥。但即使并轨，缴费基数差异也能让到手退休金差出一倍到几倍不等，而且城乡居民那边确实不好改，还不如按身份证号发消费券。美国这边社会安全金购买力跟国内企业退休金水平相当，也就维持饿不死状态，但稍微像样的企业都有401k这种退休金账户来补充，类似国内事业单位的年金。也就是说国家层面退休金更多是托底的而不是享受水平，这个角度就要把机关事业单位的退休金补贴给城乡居民，然后推广企业年金或个人退休金账户等其他养老途径，因为执行者利益相关，改革难度非常高。关于你说地方政府问题，地方公务员冗余度确实过高，但中央公务员其实还缺，可以将部分地方公务员改为中央部委直辖，脱离地方财税，这样好推行政策，但集权过多又影响行政效率，这笔账还是让喜欢操心的人算吧。

**Reply by earfanfan** [reply]
>这笔账还是让喜欢操心的人算吧。

我昨天耕博客的时候看到你的回复，当时忽然觉得很心安，也顺着你的思路就觉得你说得对呀，我操不了这个心的，还是继续无脑傻乐地过日子吧。然鹅睡了一觉起来，我的想法又有些变化，今年国内出了好多起无差别杀人事件，我隐约感觉底层不稳，不过我从小生活在农村，没有生活在城市不知道以前城市治安怎么样，但听我弟讲他跟着我爸妈零几年出来打工的时候社会上男盗女娼，各种偷鸡摸狗的事情非常猖獗，据说我爸有一次晚上骑车去上夜班经过一条小巷子看到有人互砍、鲜血飞溅，现在是看不到这样的事情的，所以他说治安变好了，我也相信。

不过我昨天耕了一篇[看一看我国刑事案件数据](https://yuanfan.rbind.io/project/court-data/)，这是我昨天摸鱼的时候搜集的官方数据，看起来刑事案件数量整体变多了，如果只看刑事犯罪类型的占比的话，侵犯公民人身权利民主权利罪的占比下降了，而危害公共安全罪和妨害社会管理秩序罪的占比上升了。如果我对此不负责任地瞎猜的话，就是针对个人的寻仇或坑害变少了，但报复社会的变多了。这个数据看上去好像真的是底层不稳了。

所以我又想啊，住房、医疗、教育、养老这些方面有很多社会问题，每个都对老百姓影响很大，但对上面来说到底哪个问题更紧迫我们是不知道的，也许解决这些问题的重点并不是答案是什么，而是问题到了怎样严重的地步到了什么样不得不解决的时机，才会倒逼上面下定决心去改革。

在我至今为止接触过的信息里面，曾经有一个这样的时机。我之前算过1989年至2005年农业税占我国各项税收之和的比例，范围在3%到6%之间，平均值为3.8%。但要到2006年1月1日才正式全面取消农业税。这中间的曲折我以前好像在你的哪篇博客下写过，这次不重复了。[2002年我国有20个省开始实行农村税费改革](https://www.gov.cn/gongbao/content/2003/content_62076.htm)，改革的重点是减轻农民负担，但2002年的农业税税额却比上一年增加了49%。交税的农民和征税的基层干部矛盾继续激化。在之后的几年时间里就出现了一个时机，终于倒逼上面下定决心了。很多人理所当然以为是农村的底层不稳起到了最大的作用，但有另一种我更认同的说法是，其实是基层干部层面的底层不稳起到了最大的作用。

我看县级公报的文本和数据时，看到很多县的机关事业单位早在好些年前就出现了养老金缺口，但各县政府都选择了用财政收入来补贴，保障机关事业单位的养老金足额发放。我忍不住猜想，这其实也是上级默许的，因为保障政府的运行对上面来讲更重要吧，不然的话一旦基层干部调转方向，所能搅动的力量会更大。

**Reply by yufree** [reply]
报复社会的人在任何时代都有，更确切说，是有拉着陌生人一起升天这种思想在一定人口基数上是必然出现的。但经济条件越好，生活里的选项就越多，更不容易在单一思想指导下钻牛角尖，此时这种思想付诸行动的概率就越低。但经济大环境一旦下行，那种完全看不到希望的人就多一些，如果经济下行伴随贫富差距加大，那比一起受穷还麻烦。如果找不到经济增长新叙事，要么你从外部找个敌人来甩锅，要不就要内部搞公平，历史上经常是同时发生。
农村的问题终极解决方法不是什么好政策，就是消灭掉农村这种经济组织形式，大家都进城，让市民矛盾替代乡土矛盾。有些问题根本就解决不了，但只要问题中的人要考虑其他问题了或不存在了，也就不是问题了，但碰上钻牛角尖的也就只能认栽了。每个具体的问题可能都有最优解，但求解方案对人类管理者要求还是太高了，不如交给AI。

**Reply by earfanfan** [reply]
嗯，那我也不纠结了，你说得对，谢谢三水，你真好。

**Reply by earfanfan** [reply]
三水你好，我刚摸鱼的时候忽然想到一个细节上的矛盾，近二十年来我们国家的经济大环境好像（PS数据给我的感觉）整体上就是在上行，但刑事犯罪案件数是不断增多的。下面我简单算了下案件的增长率，无意中发现危害公共安全罪的案件数是在2011和2012年出现了大幅度的上涨。所以我又想了一下，如果是在社会福利完善的国家，人民生活有国家兜底，那么要经济下行贫富差距拉大才会出现更多拉陌生人升天的人。可是如果是在社会福利发展不完善的国家，国家资源有限，越是追求经济增长，越是把更多资源分配给经济建设而忽略民生建设，就越是容易把很多隐藏的问题摁下不表。当然，这是一种逻辑推理，我2011-2012年的时候还在上大学，那时候关注到的社会问题有限，完全想不起来那时候有过撒爆炸新闻了。记得你比我大5岁咧，那时候至少大学毕业了，年纪更大的人说不定关注到的社会问题多点。

|年份|危害公共安全罪收案数(件)|增长率1|妨害社会管理秩序罪收案数(件)|增长率2|
|:--:|:--:|:--:|:--:|:--:|
|2023|395664|0.13|378904|0.26|
|2022|349753|-0.2|300622|-0.09|
|2021|435951|0.19|330311|0.32|
|2020|365228|-0.12|250895|-0.19|
|2019|417017|0.22|309166|0.09|
|2018|340629|-0.02|283126|-0.2|
|2017|346856|0.28|355179|0.4|
|2016|271877|0.1|254357|-0.09|
|2015|247795|0.13|279822|0.22|
|2014|219894|0.14|228684|0.13|
|2013|193275|0.1|201688|0.1|
|2012|175439|***0.47***|183841|0.07|
|2011|119659|***0.35***|171840|0.12|
|2010|88950|0.02|153201|0.14|
|2009|86987|-0.04|134380|0.1|
|2008|90396|0.07|121901|0.15|
|2007|84795|0.07|106412|0.12|
|2006|79583|0.06|95330|0.09|
|2005|75099|0.09|87501|0.11|
|2004|68795|0.2|79073|0.04|
|2003|57369|0.07|76072|-0.01|
|2002|53433|0.04|77196|-0.02|

>每个具体的问题可能都有最优解，但求解方案对人类管理者要求还是太高了，不如交给AI。

原来你思考那么多是为了求解嘛？我思考并不是为了求解，大概只是不想看待很多问题时轻易地就选一个狭隘的角度然后结束思考，也不想找你借了答案就迁移到我的脑子里然后结束思考。

又另，我最近收集了我们国家官方发布的教育统计数据，打算分析一下，还有一些别人给我说的，我说给你，看你有撒看法冇得。一个是很多人在说中考是分流，高考才是竞争，我们公司有个比我年长10岁的姐姐跟我讲，现在武汉某个新区城市边缘的中学升高中的升学率是尽量做到30%，有些好的公立学校基本上到了60%-70%（PS她说的那个学校班级多得夸张，一个年纪35个班），而一些新建的私立学校因为掐尖的缘故升高中的升学率到了90%以上。正是因为这种卷到极致的卷法，我上高中的时候还是只到了高三才进入备战状态，而我今年上高中的堂弟已经变成了高中三年备战高考，而武汉有些学校更是夸张到了初中就开始备战高考了，当然小学生也没能幸免，作业也是超级多，做作业到晚上九十点变常态了。我之前看了一篇论文[中国教育改革为什么会这么难](https://www.aisixiang.com/data/88347.html)，大约是说整个教育系统和多方利益绑定，每每上面出好政策，下面自有应付的方法，还是卷生卷死的。

我想的是，难道没有什么办法阻止这种卷嘛？那位姐姐说之所以卷成这样，是因为她本人是上一轮教育红利的受益者，很多与她相似的人同样是不愿意下一代阶层滑落的，总想抓住教育红利的尾巴，所以要在能力范围内争取资源给孩子。不过按照现在的新生人口数的变化趋势，再过十年二十年，近几十年发展出来的教育资源自然就够后面的人使用了，是不是直接就不卷了呢。

**Comment by misaka-clover** [comment]
非常新颖的角度，感谢您的文章！

**Comment by Glanwu** [comment]
非常有思考价值的文章，特地登陆留言~
